src/moscot/problems/space/_mapping.py

```diff
@@ -299,8 +314,17 @@ def solve(
         - :attr:`solutions` - the :term:`OT` solutions for each subproblem.
         - :attr:`stage` - set to ``'solved'``.
         """
+        additonal_kwargs = {}
+        if self.problem_kind == "quadratic":
+            additonal_kwargs["alpha"] = alpha
+            additonal_kwargs["linear_solver_kwargs"] = linear_solver_kwargs
+        else:
+            if alpha != 0:
+                raise ValueError("Expected `alpha` to be 0 for a `linear problem`.")
+            additonal_kwargs.update(linear_solver_kwargs)
+            # logger.warning("Solving a `linear problem`, ignoring `alpha`.")
```

Comment:
Maybe you forgot to delete this commented line 325

Reply:
ah great, thanks!